unlock-app/src/pages/log.js

```diff
@@ -55,6 +58,15 @@ Log.propTypes = {
   account: UnlockPropTypes.account.isRequired,
   network: UnlockPropTypes.network.isRequired,
   transactionFeed: PropTypes.arrayOf(UnlockPropTypes.transaction).isRequired,
+  metadata: PropTypes.objectOf(
```

Comment:
Do you mind putting propType definition in `proptypes` with the other ones?

Reply:
agree this would be a great fit for `src/propTypes.js` as a new `transactionMetadata` prop type